internal/sapcontrol/soap_wsdl.go

```diff
@@ -0,0 +1,66 @@
+package sapcontrol
+
+import (
+	"context"
+	"encoding/xml"
+
+	"github.com/hooklift/gowsdl/soap"
+)
+
+type GetProcessList struct {
+	XMLName xml.Name `xml:"urn:SAPControl GetProcessList"`
+}
+
+type GetProcessListResponse struct {
+	XMLName xml.Name          `xml:"urn:SAPControl GetProcessListResponse"`
+	Process *ArrayOfOSProcess `xml:"process,omitempty" json:"process,omitempty"`
+}
+
+type OSProcess struct {
+	Name        string     `xml:"name,omitempty" json:"name,omitempty"`
+	Description string     `xml:"description,omitempty" json:"description,omitempty"`
+	Dispstatus  STATECOLOR `xml:"dispstatus,omitempty" json:"dispstatus,omitempty"`
+	Textstatus  string     `xml:"textstatus,omitempty" json:"textstatus,omitempty"`
+	Starttime   string     `xml:"starttime,omitempty" json:"starttime,omitempty"`
+	Elapsedtime string     `xml:"elapsedtime,omitempty" json:"elapsedtime,omitempty"`
+	Pid         int32      `xml:"pid,omitempty" json:"pid,omitempty"`
+}
+
+type ArrayOfOSProcess struct {
+	Item []*OSProcess `xml:"item,omitempty" json:"item,omitempty"`
+}
+
+type STATECOLOR string
+
+const (
+	STATECOLORGRAY   STATECOLOR = "SAPControl-GRAY"
+	STATECOLORGREEN  STATECOLOR = "SAPControl-GREEN"
+	STATECOLORYELLOW STATECOLOR = "SAPControl-YELLOW"
+	STATECOLORRED    STATECOLOR = "SAPControl-RED"
+)
+
+type WebService interface {
+	/* Returns a list of all processes directly started by the webservice according to the SAP start profile. */
+	GetProcessList() (*GetProcessListResponse, error)
+}
+
+type webService struct {
+	client *soap.Client
+}
+
+func (service *webService) GetProcessList() (*GetProcessListResponse, error) {
+	request := &GetProcessList{}
+	response := new(GetProcessListResponse)
+	err := service.client.CallContext(context.Background(), "''", request, response)
+	if err != nil {
+		return nil, err
+	}
+
+	return response, nil
+}
+
+func NewWebService(client *soap.Client) WebService {
```

Comment:
we should comment all the new exported function `golint` practice and so on

Reply:
will do!